dotCMS/src/main/java/com/liferay/portal/model/User.java

```diff
@@ -296,6 +300,23 @@ public void setModificationDate(final Date modificationDate) {
         setModified(true);
     }
 
+    public boolean isBackendUser(){

```

Comment:
test

Reply:
I didn't think a Test was required here since we would only be verifying a role is present or absent which is already covered in many spots. But you've got it. I'll add one.

Reply:
done.